CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ See [RELEASE](./RELEASE.md) for workflow instructions.
 
 ### Highlights
 
+* [#5814](https://github.com/spacemeshos/go-spacemesh/pull/5814) Removed in-code local DB migrations.
+  Updating to this version requires going through v1.4 first.
+
 ### Improvements
```

Comment:
NIT: `### Improvements` is normally the last section of the changelog of a release (see `RELEASE.md` in the repo)